src/AutoRest.CSharp/LowLevel/Output/OperationMethodChainBuilder.cs

```diff
@@ -305,6 +305,13 @@ private IEnumerable<Parameter> BuildSpreadParameters(ModelTypeProvider model)
 
         private void BuildParameters()
         {
+            SerializationFormat GetSerializationFormat(InputParameter parameter)
```

Comment:
Can you explain why we need a check on `serializationFormat`?

Reply:
It is not a check. It is getting the serialization format. And it comes from Dapeng's comment that extract that logic into a function.